exercises/kindergarten-garden/canonical-data.json

```diff
@@ -0,0 +1,127 @@
+{
+  "exercise": "kindergarten-garden",
+  "version": "1.0.0",
+  "comments": [
+    "Garden can be created with just a diagram or a diagram and students.",
+    "Students' gardens are assigned in alphabetical order.",
+    "The default students are:",
+    "  Alice, Bob, Charlie, David, Eve, Fred, Ginny, Harriet, Ileana, Joseph, Kincaid, and Larry."
```

Comment:
Ah, that does make sense. Were you referring to all 4 lines? I included the bit about the default students because I thought it might help clarify some of the issues brought up in #659. Would that still be considered unnecessary though?